File-Formats/PDF/Convert-HTML-To-PDF/blazor.md

```diff
@@ -9,116 +9,124 @@ keywords: Assemblies
 
 # Convert HTML to PDF file in Blazor
 
-Steps to convert HTML to PDF in Blazor application
+The Syncfusion HTML to PDF converter is a .NET library used to convert HTML or web pages to PDF document in Blazor.
 
-Create a new C# Blazor application project. Select Blazor App from the template and click the Next button.
+## Steps to convert HTML to PDF in Blazor application
+
+1. Create a new C# Blazor Server application project. Select Blazor App from the template and click the Next button.
 ![Blazor_step1](htmlconversion_images/blazor_step1.png)
 
-Now, the project configuration window appears. Click Create button to create a new project with the default project configuration.
+   In the project configuration window, name your project and select Create.
 ![Blazor_step2](htmlconversion_images/blazor_step2.png)
 
-Install the [Syncfusion.HtmlToPdfConverter.Net.Windows](https://www.nuget.org/packages/Syncfusion.HtmlToPdfConverter.Net.Windows/) NuGet package as a reference to your Blazor application from [NuGet.org](https://www.nuget.org/).
+2. Install the [Syncfusion.HtmlToPdfConverter.Net.Windows](https://www.nuget.org/packages/Syncfusion.HtmlToPdfConverter.Net.Windows/) NuGet package as a reference to your Blazor Server application from [NuGet.org](https://www.nuget.org/).
 ![Blazor_step3](htmlconversion_images/blazor_step3.png)
 
-Create a new cs file named ExportService under Data folder and include the following namespaces in the file.
-
-{% highlight c# tabtitle="C#" %}
+3. Create a new class file named ExportService under Data folder and include the following namespaces in the file.
 
-using Syncfusion.HtmlConverter;
-using Syncfusion.Pdf;
-using System.IO;
+   {% highlight c# tabtitle="C#" %}
 
-{% endhighlight %}
+   using Syncfusion.HtmlConverter;
+   using Syncfusion.Pdf;
+   using System.IO;
 
+   {% endhighlight %}
 
-Add the following method in the ExportService class
+4. Add the following code in the ExportService class.
```

Comment:
Add the following code to convert HTML to PDF document in ExportService class.

Reply:
content changed